src/legacy/core_plugins/kibana_react/public/markdown/markdown.tsx

```diff
@@ -43,7 +51,8 @@ export const markdownFactory = memoize(
       markdownIt = new MarkdownIt('zero', { html: false, linkify: true });
       markdownIt.enable(whiteListedRules);
```

Comment:
It looks like `whitelistedRules` isn't actually in use today. That parameter can be used to configure the Markdown renderer to only allow a subset of Markdown syntax.
I figured if anyone wants to use this, they don't want to allow HTML elements to be rendered either. However, if we need a separate parameter to enable/disable the whitelisted HTML elements from rendering, we can certainly add that.